charts/charts.mk

```diff
@@ -1,97 +1,66 @@
 EMISSARY_CHART = $(OSS_HOME)/charts/emissary-ingress
-thisdir := $(patsubst %/,%,$(dir $(lastword $(MAKEFILE_LIST))))
-
-
-define _push_chart
-	CHART_NAME=$(1) $(OSS_HOME)/charts/scripts/push_chart.sh
-endef
-
-define _set_tag_and_repo
-	$(OSS_HOME)/venv/bin/python $(OSS_HOME)/charts/scripts/update_chart_image_values.py \
-		--values-file $(1) --tag $(2) --repo $(3)
-endef
-
-define _set_tag
-	$(OSS_HOME)/venv/bin/python $(OSS_HOME)/charts/scripts/update_chart_image_values.py \
-		--values-file $(1) --tag $(2)
-endef
 
 push-preflight: create-venv $(tools/yq)
 	@$(OSS_HOME)/venv/bin/python -m pip install ruamel.yaml
 .PHONY: push-preflight
 
 release/ga/chart-push:
 	$(OSS_HOME)/releng/release-wait-for-commit --commit $$(git rev-parse HEAD) --s3-key chart-builds
-	for chart in $(EMISSARY_CHART) ; do \
-		$(call _push_chart,`basename $$chart`) ; \
-	done ;
+	CHART_NAME=$(notdir $(EMISSARY_CHART)) $(OSS_HOME)/charts/scripts/push_chart.sh
 .PHONY: release/ga/chart-push
 
 release/promote-chart-passed:
 	@set -ex; { \
-		commit=$$(git rev-parse HEAD) ;\
-		printf "$(CYN)==> $(GRN)Promoting $(BLU)$$commit$(GRN) in S3...$(END)\n" ;\
-		echo "PASSED" | aws s3 cp - s3://$(AWS_S3_BUCKET)/chart-builds/$$commit ; \
+	  commit=$$(git rev-parse HEAD); \
+	  printf "$(CYN)==> $(GRN)Promoting $(BLU)$$commit$(GRN) in S3...$(END)\n"; \
+	  echo "PASSED" | aws s3 cp - s3://$(AWS_S3_BUCKET)/chart-builds/$$commit; \
 	}
 .PHONY: release/promote-chart-passed
 
 chart-push-ci: push-preflight
-	if [ ! -z $(IS_PRIVATE) ]; then \
-		echo "Private repo, not pushing chart" ;\
-		exit 1 ;\
-	fi;
+	[[ -z "$(IS_PRIVATE)" ]] || (echo "Private repo, not pushing chart" >&2; exit 1)
 	@[ -n "${CHART_VERSION_SUFFIX}" ] || (echo "CHART_VERSION_SUFFIX must be set for non-GA pushes" && exit 1)
 	@[ -n "${IMAGE_TAG}" ] || (echo "IMAGE_TAG must be set" && exit 1)
 	@[ -n "${IMAGE_REPO}" ] || (echo "IMAGE_REPO must be set" && exit 1)
-	set -e; { \
-		for chart in $(EMISSARY_CHART) ; do \
-			sed -i.bak -E "s/version: ([0-9]+\.[0-9]+\.[0-9]+).*/version: \1${CHART_VERSION_SUFFIX}/g" $$chart/Chart.yaml && rm $$chart/Chart.yaml.bak ; \
-			$(call _set_tag_and_repo,$$chart/values.yaml,${IMAGE_TAG},${IMAGE_REPO}) ; \
-			$(tools/yq) w -i $$chart/Chart.yaml 'appVersion' ${IMAGE_TAG} ; \
-			$(call _push_chart,`basename $$chart`) ; \
-		done ; \
-	}
+	sed -i.bak -E "s/version: ([0-9]+\.[0-9]+\.[0-9]+).*/version: \1${CHART_VERSION_SUFFIX}/g" $(EMISSARY_CHART)/Chart.yaml && rm $(EMISSARY_CHART)/Chart.yaml.bak
+	{ $(OSS_HOME)/venv/bin/python $(OSS_HOME)/charts/scripts/update_chart_image_values.py \
+	  --values-file=$(EMISSARY_CHART)/values.yaml \
+	  --tag=$(IMAGE_TAG) \
+	  --repo=$(IMAGE_REPO); }
```

Comment:
What's the value in the `{}` here? Just visual cues for what the command really is?

Reply:
So it'll tell me what the mistake is if I forget a `\` at the end of a line.